### PR TITLE
Fixed missing dependency and shell lint check

### PR DESCRIPTION
**Pull request by naveensrinivasan**
* Fixed the missing dependency go-licenses
   * Fixed the shell lint check

### CODE REVIEW

File: tools.go [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tools
+
+import (
+	_ "github.com/google/go-licenses/licenses"
```
Comment (priyawadhwa) [inline tools.go:18]: I think we can put this in this file instead:

https://github.com/tektoncd/chains/blob/main/hack/tools.go

Comment (naveensrinivasan) [reply tools.go]: @priyawadhwa  Thanks moved it.

**Comment by naveensrinivasan** [comment]
ping @priyawadhwa, let me know if something else is missing. Thanks

**Pull request merged**